doc/RELEASE_PROCESS.rst

```diff
@@ -78,69 +74,47 @@ This is an outline of what needs to be done in order to release rst2pdf.
     ::
 
        $ echo -e "[egg_info]\ntag_build=rc1\n" > /tmp/build_opts.cfg
-       $ DIST_EXTRA_CONFIG=/tmp/build_opts.cfg python -m build
+       $ DIST_EXTRA_CONFIG=/tmp/build_opts.cfg uv build
        $ rm /tmp/build_opts.cfg
 
     If you're doing an alphaX, betaX or postX, then change ``tag_build=rc1`` appropriately
 
-#. Set up PyPI if you haven't already
-
-    Create a ``~/.pypirc`` file with sections for rst2pdf and testrstpdf which are then used with ``twine`` via the
-    ``--repository`` switch.
-
-    It should contain the following:
-
-    ::
-
-        [rst2pdf]
-          username = __token__
-          password = {your token here}
-
-        [testrst2pdf]
-          repository = https://test.pypi.org/legacy/
-          username = __token__
-          password = {your token here}
-
-
-    You can get your token from the account settings section of https://pypi.org/ & https://test.pypi.org/
-
-
-#. Upload the rc distribution to Test-PyPI_
+#. Upload the rc distribution to Test-PyPI_. You can get your token from the account settings section of
+   https://test.pypi.org/.
 
     ::
 
-       $ twine upload --repository testrst2pdf dist/*
+       $ uv publish --publish-url https://test.pypi.org/legacy/ --token {your token here}
 
     Check that it all looks correct on Test-PyPI. If not, fix and release a new rc.
 
-#. Test Test-PyPI release into a clean virtual env
+#. Test Test-PyPI release into a clean virtual env (specify the correct version number below). It should install the
+   rc release on Test PyPI and be able to create PDF documents from rst files e.g.
 
     ::
 
-       $ pip install --index-url https://test.pypi.org/simple \
-         --extra-index-url https://pypi.org/simple --pre rst2pdf
-
-    It should install the rc release on Test PyPI and be able to create PDF documents from rst files
+       $ uvx -n --index-url https://test.pypi.org/simple --extra-index-url https://pypi.org/simple --prerelease allow \
+         --index-strategy unsafe-best-match rst2pdf@0.101rc1 tests/input/test_tableofcontents.rst
 
-    Delete the build artifacts and dist files with:
+#. Delete the build artifacts and dist files with:
 
     ::
 
-        $ rm -rf build/ rst2pdf.egg-info/ dist/
+       $ rm -rf build/ rst2pdf.egg-info/ dist/
 
 #. Once rc version is working, release to PyPI_ by generating official release and uploading
 
     ::
 
-       $ python -m build
-       $ twine upload --repository rst2pdf dist/*
+       $ uv build --no-sources
+       $ uv publish --token {your token here}
```

Comment:
Is there a way to store the token somewhere such that it doesn't appear in the bash history?

Reply:
I believe it can be placed in an environment variable (`UV_PUBLISH_TOKEN`), would that be better? Depending on how that's set locally for the release process, it should be an improvement on storing the old token in a plaintext config file `~/.pypirc`

https://docs.astral.sh/uv/guides/publish/#publishing-your-package

Reply:
One day, we should automate publishing to PyPI when a tag is pushed to git :)

Reply:
Let's leave this as it is for now - I think @akrabat is the only person to have published in recent memory, although in theory I also have access. It's fine as it is or we can adjust it whenever we next release and need these instructions.